### PR TITLE
Update-Ca-Certificates from LWAS 

### PR DESCRIPTION
**Pull request by CooperLink**
As of Antares 101, LWAS allows customer to upload client certificates that it will reflect into the customer's container.  This PR adds a startup script that uses update-ca-certificates to update the cert available in the customer's container with the cert provided by LWAS. 

Websites PR : https://msazure.visualstudio.com/One/_git/AAPT-Antares-Websites/pullrequest/8466071?_a=files
This PR also removes un-used start.sh and sshd_config files in favor of the ones in sharedconfig that are distributed to all appservice images that use them. 

Finally, introduce a new file start_nonappservice.sh to provide an extension point for future scripts if necessary.  This script will be used to run local shell files before starting the Functions Host

Testing locally and on early Antares regions using the resulting images from this PR build 
[] - Containers build and start successfully
[] - Custom Images on LWAS use the new script to install a custom ca-certificate

### PR information

- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/development-docs/cleaning-up-commits.md).
- [x] If applicable, the PR references the bug/issue that it fixes in the description.
- [] New Unit tests were added for the changes made and CI is passing.

### CODE REVIEW

**Comment by CooperLink** [comment]
I tested this locally by generating a ca certificate and running the container.  The result below shows updated installation of the ca-certificates
image

**Comment by CooperLink** [comment]
Prod test was successful

image

File: host/4/bullseye/amd64/node/node16/node16-slim.Dockerfile [hunk 1]
```diff
@@ -58,5 +58,12 @@ RUN apt-get update && \
 COPY --from=runtime-image [ "/azure-functions-host", "/azure-functions-host" ]
 COPY --from=runtime-image [ "/workers/node", "/azure-functions-host/workers/node" ]
 COPY --from=runtime-image [ "/FuncExtensionBundles", "/FuncExtensionBundles" ]
+COPY install_ca_certificates.sh start_nonappservice.sh /opt/startup/
+RUN chmod +x /opt/startup/install_ca_certificates.sh && \
+    chmod +x /opt/startup/start_nonappservice.sh
 
-CMD [ "/azure-functions-host/Microsoft.Azure.WebJobs.Script.WebHost" ]
+RUN chmod +x /opt/startup/install_ca_certificates.sh && \
```
Comment (zahalzel) [inline host/4/bullseye/amd64/node/node16/node16-slim.Dockerfile:63]: Looks like a duplicate

**Review by zahalzel — DISMISSED** [review]
Looks good, just one minor duplicated section

File: host/4/bullseye/amd64/node/node16/node16-slim.Dockerfile (hunk 1 above)
Comment (CooperLink) [reply host/4/bullseye/amd64/node/node16/node16-slim.Dockerfile:63]: Thanks for callout fixed.

**Review by CooperLink — COMMENTED** [review]
reviewed each file

**Review by Tonewall — DISMISSED** [review]

**Review by pragnagopa — DISMISSED** [review]

**Comment by CooperLink** [comment]
Validated on bookworm also. Got "skipping ca-certificates" but still succeeded in setting up the ca-cert. 
image

**Review by Tonewall — APPROVED** [review]

**Review by zahalzel — APPROVED** [review]
Looks good

**Pull request merged**